msprime/trees.py

```diff
@@ -684,7 +684,7 @@ def time(self, u):
     def get_time(self, u):
         """
         Returns the time of the specified node in generations. Returns 0 if u
-        is a leaf or is not a node in the current tree.
+        is a sample or is not a node in the current tree.
```

Comment:
I think that this should read "Returns 0 if u is not a node in the current tree."

Reply:
Well spotted. Opening an issue on this.